### ISSUE TITLE
Narrator doesn't distinguish group headers from channels

### ISSUE THREAD

**Issue by jaigak** [BreeceW/Oxidized#110]
**Describe the bug**
A clear and concise description of what the bug is and, if applicable, what you think should happen instead.
Currently, screen readers such as the Narrator doesn't distinguish group headers from channels so it can be confusing. For clarity, say {Name} group header instead of just {Name}. Eg: ToString will look like $"{Name}, group header".

**To Reproduce**
1. Open Narrator or any other screen reader.
2. Focus a group header and then do the same with a channel item. There's nothing to distinguish channels from group headers.

**System(s) that problem occurred on**
Windows 11 Pro Insider Preview 22000.71
PC
x64

**Comment by jaigak** [comment]
Also, it seems like all the groups are in a single list with a single tab stop. It would be better if they were separate so you can easily switch between groups without switching to other channels first through the keyboard but it could be tricky to implement since it's not like a tree view nor semantic zoom.

**Comment by BreeceW** [comment]
> Also, it seems like all the groups are in a single list with a single tab stop. It would be better if they were separate so you can easily switch between groups without switching to other channels first through the keyboard but it could be tricky to implement since it's not like a tree view nor semantic zoom.

Can you point to an example of an app with a similar hierarchy that demonstrates how you think the keyboard behavior should work? It seems fine to me.

**Comment by jaigak** [comment]
In the hierarchical nav view, the tab key only goes through the first item or the main item and to navigate inside a group, you need to use arrow keys however in Oxidized, arrow keys go through all channels, not just in the current group and you can't tab to switch to the next group as the group header itself is treated as a channel.

**Comment by BreeceW** [comment]
> In the hierarchical nav view, the tab key only goes through the first item or the main item and to navigate inside a group, you need to use arrow keys however in Oxidized, arrow keys go through all channels, not just in the current group and you can't tab to switch to the next group as the group header itself is treated as a channel.

I don’t think that behavior makes sense for the channel list. It seems like it would create a lot of unnecessary tabbing, especially since the channel list is user-generated and can be arbitrarily long and most of the time, most items will be expanded.
